Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -646,6 +646,13 @@ extension SPNoteListViewController {
 
     @objc(keyboardWillChangeFrame:)
     func keyboardWillChangeFrame(note: Notification) {
+        
+        // No window means we aren't in the view hierarchy.
+        if self.view.window == nil {
```

Comment:
When the SPNoteListViewController class tries to handle this notification while not currently onscreen, it causes warnings in the console about laying out views/cells with potentially invalid layout data (rough summary). The basic idea from the console error is that we shouldn't be doing view layout while not in the view hierarchy, so this test prevents that from happening.

These console errors are pretty easily missed thanks to all the logging from Simperium but they should be there on the `develop` branch.

The other option would be to update SPNoteListViewController to only listen for notifications when onscreen

Reply:
This is the warning from `develop`:

![Console-Warning](https://user-images.githubusercontent.com/40267301/81675699-c8df7080-9481-11ea-84c0-cf925cad78de.png)

And the full console log message:

```
2020-05-12 18:50:24.329016+0800 Simplenote[62220:5302818] [TableView] Warning once only: UITableView was told to layout its visible cells and other contents without being in the view hierarchy (the table view or one of its superviews has not been added to a window). This may cause bugs by forcing views inside the table view to load and perform layout without accurate information (e.g. table view bounds, trait collection, layout margins, safe area insets, etc), and will also cause unnecessary performance overhead due to extra layout passes. Make a symbolic breakpoint at UITableViewAlertForLayoutOutsideViewHierarchy to catch this in the debugger and see what caused this to occur, so you can avoid this action altogether if possible, or defer it until the table view has been added to a window. Table view: <UITableView: 0x7f8bc7819400; frame = (0 0; 375 667); clipsToBounds = YES; gestureRecognizers = <NSArray: 0x60000210e730>; layer = <CALayer: 0x600002f5e100>; contentOffset: {0, -120}; contentSize: {375, 348}; adjustedContentInset: {120, 0, 260, 0}; dataSource: <SPNoteListViewController: 0x7f8bd5415dc0>>
```

Reply:
Done and updated!